src/components/IOUConfirmationList.js

```diff
@@ -339,56 +303,68 @@ class IOUConfirmationList extends Component {
                 }
                 return participant;
             });
-            return {participants: newParticipants};
+            const newSelectedParticipants = _.where(newParticipants, {selected: true});
+            const newSections = this.getSections(newParticipants);
+
+            return {
+                sections: newSections,
+                participants: newParticipants,
+                selectedParticipants: newSelectedParticipants,
```

Comment:
NAB, I'm a little confused about this change and why we need to track all three of these separately where the `participants` was enough before. It kinda seems like we are just duplicating very similar information in state instead of deriving it from one state object.

Reply:
Maybe there is a reason it needed to happen in this PR but I'm not sure if it improved things otherwise.

Reply:
Yeah, I think I kinda just did this based on what I perceive as some inefficiencies/React anti-patterns, but without benchmarking I don't really know if it makes any real difference. In any event, it doesn't need to happen in this PR, so I removed it now and the diff in this file is smaller